### PR TITLE
ROC-3803: Adding javascript for legal FE

### PR DESCRIPTION
**Pull request by singh-surjeet**
### Story:
https://tools.hmcts.net/jira/browse/ROC-3803

### Description:
Disabling submit button on claim/response journey to prevent multiple submits. 

### Notes:
After successfully testing on citizen FE, replicating it on legalRep.

### CODE REVIEW

File: src/main/views/includes/scripts.njk [hunk 1]
```diff
@@ -7,6 +7,8 @@
 <script src="{{ asset_paths['js'] }}/detectJavascript.js"></script>
 <script src="{{ asset_paths['js'] }}/analytics.js"></script>
 <script src="{{ asset_paths['js'] }}/jquery-prevent-double-submit.js"></script>
```
Comment (singh-surjeet) [inline src/main/views/includes/scripts.njk]: @hmcts/cmc 
We have one jquery implementation targeting page pay-by-account.njk - src/main/features/claim/views/pay-by-account.njk implemented a whilst back. It doesn't disable button but submit page once.

File: src/main/views/includes/scripts.njk [hunk 2]
```diff
@@ -7,6 +7,8 @@
...
+<script src="{{ asset_paths['js_vendor'] }}/nodelist-foreach-polyfill.js"></script>
```
Comment (geek-soft) [inline src/main/views/includes/scripts.njk]: Have you checked whether file exists?

File: src/main/views/includes/scripts.njk (hunk 1 above)
Comment (geek-soft) [reply src/main/views/includes/scripts.njk]: Remove old jquery based solution then. Supporting both is not an option.

File: package.json [hunk 3]
```diff
@@ -97,7 +97,8 @@
     "ts-node": "^7.0.0",
     "tsconfig-paths": "^3.1.1",
     "typescript": "^2.4.2",
-    "url-parse": "^1.2.0"
+    "url-parse": "^1.2.0",
+    "nodelist-foreach-polyfill": "^1.2.0"
```
Comment (geek-soft) [inline package.json]: List of packages should be in alphabetical order. Please move into right position.

File: src/main/public/js/double-click-preventer.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+            element.disabled = true
+            element.form.submit()
+          }
+        })
+      })
+  })
+
+})()
+
+
```
Comment (geek-soft) [inline src/main/public/js/double-click-preventer.js]: Remove excessive blank line.

File: package.json (hunk 3 above)
Comment (singh-surjeet) [reply package.json]: Well spotted, addressed.

File: src/main/public/js/double-click-preventer.js (hunk 4 above)
Comment (singh-surjeet) [reply src/main/public/js/double-click-preventer.js]: Done.

**Comment by singh-surjeet** [comment]
Testing has finished successfully. Needed to merge both FE branches.

**Pull request merged**